### ISSUE TITLE
-seed option?

### ISSUE THREAD

**Issue by jbgrant01** [jcjohnson/fast-neural-style#23]
I am playing around with your code to process a directory of frames from a video. I have the code in a Bash shell for WIN10 and even without CUDA support it is blazingly fast. Nice work! I am getting a lot of artifacts in the background. When I was doing a similar process with your original neural-style there was a -seed option that seemed to smooth out the frame to frame differences.

I was wondering if it would be difficult to have a similar option in fast-neural-style?

I have used Manual Ruder et al artistic-videos code, but my system isn't up to the task without it taking days of processing. It does produce nice results using the optical flow and temporal flow. I did get nearly the same results when processing with the -seed option in neural-style without the optical flow.

**Comment by jcjohnson** [comment]
The original neural-style generates random noise to initialize the generated image; setting the random seed with -seed ensures that the same random image is used to initialize each video frame. This project works very differently - it generates stylized images by processing the content images with a trained neural network. This feedforward processing does not involve generating any random numbers, so setting the random seed will not do anything.

In short, this means that there is no simple way to add a -seed option that does what you want.
